plugins/woocommerce-blocks/assets/js/atomic/blocks/product-elements/sku/style.scss

```diff
@@ -3,7 +3,16 @@
 	@include font-size(small);
```

Comment:
I tested TT5, TT4, TT3 and TT2. On TT2 the size differs.

<img width="523" alt="image" src="https://github.com/user-attachments/assets/6b002f0d-6466-4735-bc43-98af42447808">

With the current `wp-block-post-terms` class this line is no longer needed as theme takes care of the size (but due to specificity this line takes precedence over theme styles and small font-size is applied).